src/image/image.js

```diff
@@ -341,4 +341,27 @@ p5.prototype._makeFrame = function(filename, extension, _cnv) {
   frames.push(thisFrame);
 };
 
+/**
+ * Returns the current canvas as a loaded p5.Image. This allows an assignment
+ * or immediate use in the users sketch as a canvas screenshot.
+ *
+ * @method getCanvasImage
+ * @param none
+ *
+ * @example
+ * // This example shows how a user can directly create a screenshot of his
+ * // canvas and have it saved for future use or use immediately
+ * draw() {
+ *   // .. other code
+ *   var screenshot = getCanvasImage();
+ *   image(screenshot, 10, 10, 50, 50);
+ * }
+ *
+ */
+p5.prototype.getCanvasImage = function() {
+  if (this._curElement && this._curElement.elt) {
+    return loadImage(this._curElement.elt.toDataURL('image/png'));
```

Comment:
This is the part Travis is complaining about. `loadImage` is not defined in this scope and you should be accessing it via `this.loadImage`. However, I feel this implementation is a bit clunky as you are converting the canvas buffer into a png image then loading it in via `loadImage`, I wonder if it is possible to do a more direct conversion from the canvas buffer to an image buffer without this convert to a png image bit.